src/media/js/site/containers/landing.js

```diff
@@ -43,7 +45,7 @@ export class Landing extends React.Component {
               Submit a Firefox OS Add-On &rsaquo;
             </ReverseLink>
           </section>
-          {!this.props.user.token &&
+          {isLoggedOut &&
```

Comment:
`this.props.user` isn't required, so when it wasn't populated, a check for `this.props.user.token` would fail. Minor bug, test helped me find this :+1: